docs/lib/auth/fragments/ios/signin_next_steps/20_confirm_sms_mfa.md

```diff
@@ -0,0 +1,41 @@
+If you get `confirmSignInWithSMSMFACode`, the next step is to verify the user through SMS multi-factor authentication. `AuthCodeDeliveryDetails` will contain information regarding where the code has been send. You should ask the user to provide the code that they received via SMS and invoke the `confirmSignIn` api with the SMS code:
+
+<amplify-block-switcher>
+
+<amplify-block name="Listener (iOS 11+)">
+
+```swift
+func confirmSignIn() {
+    Amplify.Auth.confirmSignIn(challengeResponse: "<confirmation code received via SMS>") { result in
```

Comment:
Here and throughout in other code samples: you can make the usage a bit clearer:
```suggestion
func confirmSignIn(confirmationCodeFromUser: String) {
    Amplify.Auth.confirmSignIn(challengeResponse: confirmationCodeFromUser) { result in
```

Reply:
Updated